sets.json

```diff
@@ -461,6 +461,17 @@
     "position": -99,
     "size": -99
   },
+  {
+    "id": "216",
+    "name": "Virtual Set 16",
+    "gempName": "Set 16",
+    "abbr": "V16",
+    "legacy": false,
+    "cycle_code": "full",
+    "date_release": "2019-09-3",
```

Comment:
Although it feels like the same year is just never ending, it's actually **2021**